### ISSUE TITLE
Raspberry pi zero w hangs on boot for multiple projects

### ISSUE THREAD

**Issue by jimjsong** [nerves-project/nerves_examples#155]
### Environment

* Elixir version (`elixir -v`):
Erlang/OTP 23 [erts-11.0.2] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [hipe] [dtrace]

Elixir 1.10.3 (compiled with Erlang/OTP 22)

* Nerves environment: (`mix nerves.env --info`)
mix

|nerves_bootstrap| Environment Package List

  Pkg:         nerves_system_br
  Vsn:         1.12.0
  Type:        system_platform
  BuildRunner: {nil, []}

  Pkg:         nerves_system_rpi0
  Vsn:         1.12.1
  Type:        system
  BuildRunner: {Nerves.Artifact.BuildRunners.Docker, []}

  Pkg:         nerves_toolchain_armv6_rpi_linux_gnueabi
  Vsn:         1.3.2
  Type:        toolchain
  BuildRunner: {Nerves.Artifact.BuildRunners.Local, []}

  Pkg:         nerves_toolchain_ctng
  Vsn:         1.7.2
  Type:        toolchain_platform
  BuildRunner: {nil, []}

|nerves_bootstrap| Loadpaths Start

Nerves environment
  MIX_TARGET:   rpi0
  MIX_ENV:      dev

|nerves_bootstrap| Environment Variable List
  target:     rpi0
  toolchain:  /Users/jsong10/.nerves/artifacts/nerves_toolchain_armv6_rpi_linux_gnueabi-darwin_x86_64-1.3.2
  system:     /Users/jsong10/.nerves/artifacts/nerves_system_rpi0-portable-1.12.1
  app:        /Users/jsong10/Projects/elixir/2_nerves_examples/blinky

|nerves_bootstrap| Loadpaths End

* Additional information about your host, target hardware or environment that
  may help
running macosx 10.15.5

### Current behavior
Single Raspberry shows up when pi zero is powered on.  underscore blinking curse is underneath and stays that way.  

The same projects work fine for raspi4b.  Same sd card.  When using `export MIX_TARGET=rpi0`, the projects don't work.

Raspberrian OS can be put on the sd card and the raspberry pi boots into linux with no issues.

**Comment by jjcarstens** [comment]
@jimjsong is this an image you are seeing while using HDMI? If so, that is expected because HDMI console is disabled for the raspberry pi and expects to use the serial pins

https://github.com/nerves-project/nerves_system_rpi0/blob/main/rootfs_overlay/etc/erlinit.config#L8

Are you able to ping `nerves.local` or ssh to the rpi0?

**Comment by jimjsong** [comment]
@jjcarstens Wow...thanks for the quick response.  I tried the https://github.com/nerves-project/nerves_system_rpi0/ but get an error of:
`** (Mix)   rel/vm.args needs to be moved to rel/vm.args.eex`

In regard to pinging nerves.local or ssh to the rpi0, neither worked.  

I'm new at all this, so it's probably something I didn't setup correctly.

I did set my environment variable for my NERVES_NETWORK_PSK and NERVES_NETWORK_SSID environment variables.

Again my rpi4 works fine.  I have the hello phoenix project working on the rpi4.
